webrecorder/webrecorder/models/usermanager.py

```diff
@@ -604,7 +605,8 @@ def delete_user(self, username):
         # remove user and from all users table
         del self.all_users[username]
 
-        self.get_session().delete()
+        if self.get_session():
+            self.get_session().delete()
 
```

Comment:
Similar to above, i'd be concerned why the session would not exist at all
I assume its the command line usage, otherwise would indicate a larger problem

Reply:
Yes, its the same case as above.